src/09-workorders/work-orders.apib

```diff
@@ -215,6 +217,7 @@ Update a draft work order.
         + materials (array, optional)
             + (object)
                 + quantity: `2.2` (number, required)
+                + purchase_price (Money, optional)
```

Comment:
B: unless we decide it shouldn't be possible to unset the custom purchase price, but it makes sense in the overwritten purchase price scenario.
```suggestion
                + purchase_price (Money, optional, nullable)
```

Reply:
But it's a collection, we just overwrite the collection completely in updates, right? 🤔

Reply:
Hmm in that sense you're right indeed.